kustomization.yaml

```diff
@@ -11,3 +11,20 @@ resources:
  - storage/overlays/gcp
 configurations:
  - configs/varreference.yaml
+configMapGenerator:
+ - name: onepanel
+   namespace: onepanel
+   envs:
+     - ./vars/workflow-config-map.env
+     - ./vars/onepanel-config-map.env
+     - ./vars/logging-config-map.env
+generatorOptions:
+  disableNameSuffixHash: true
+vars:
+ - name: defaultNamespace
+   objref:
+    kind: ConfigMap
+    name: onepanel
+    apiVersion: v1
+   fieldref:
+    fieldpath: data.onepanelDefaultConfigMapNamespace
```

Comment:
@aleksandrmelnikov a bit confused about this one, this only has `fieldpath: data.onepanelDefaultConfigMapNamespace` but I see that there are more fields in the generated `ConfigMap`:

```
apiVersion: v1
data:
  loggingElasticSearchConfigMapImage: docker.elastic.co/elasticsearch/elasticsearch:7.2.0
  loggingElasticSearchConfigMapVolumeStorage: 100Gi
  onepanelDefaultConfigMapNamespace: rushtehrani
  workflowArtifactRepositoryConfigMapBucket: mlpipeline.onepanel.io
  workflowArtifactRepositoryConfigMapEndpoint: s3.amazonaws.com
  workflowArtifactRepositoryConfigMapInsecure: "false"
kind: ConfigMap
metadata:
  name: onepanel-k2mbg5b884
```

1. Where are the other fields coming from?
2. Note the namespace is missing.
3. Note the name has a hash